composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "ingenerator/scheduled-task-runner",
+    "type": "library",
+    "require": {
+        "php": "~8.2.0",
+        "ext-pcntl": "*",
+        "ext-pdo": "*",
+        "dragonmantank/cron-expression": "^3.3.2",
+        "ingenerator/php-utils": "^2.0",
+        "psr/log": "^1.0 || ^2.0",
```

Comment:
If we're not implementing a logger we could probably support ^3.0 as well?

Reply:
See https://github.com/ingenerator/scheduled-task-runner/actions/runs/7795908360/job/21259631904 I didn't think it was possible to support 1 & 3 simultaneously with the stub logger / printers we have for the tests.

Would probably be better to try and support 2 & 3 rather than 1 & 2 though

Reply:
I *think* it would work so long as you add the :void to the method signature - because an implementation can be less specific about method params, so omitting them would be ok,
 but has to be at least as specific about return.

Reply:
Oh nice, thanks for that. I hadn't realised you could do that.

Reply:
Nor had I properly till I ran into a similar thing last night!